website/src/app/views/about/members.js

```diff
@@ -539,6 +540,12 @@ const members = {
       linkedinURL: 'https://www.linkedin.com/in/rileychampion/',
       imageURL: rileychampion,
     },
+    {
```

Comment:
please move your member info to be in alphabetical by first name in the developer section. (After Rithvik Tirumala)

Reply:
gotcha, should be fixed now